Runtime/SampleHelpers/HandPose.cs

```diff
@@ -0,0 +1,31 @@
+﻿using UnityEngine;
+using UnityEngine.XR;
+
+namespace Unity.Labs.SuperScience
+{
+    /// <summary>
+    /// Helper class to map device position and rotation to the gameobject holding this component
+    /// </summary>
+    public class HandPose : MonoBehaviour
+    {
+        [SerializeField]
```

Comment:
```suggestion
#if UNITY_2019_1_OR_NEWER
        [SerializeField]
```

To fix compile errors in older versions of Unity. I was thinking about using the TPD API here as a fallback but decided it's not worth it.